### DISCUSSION TITLE
DPH dosages when it comes to combinations

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#59] [Combination Chart]
Hi there,

I've talked about this on the discord but would like to have the conversation posted here as well in addition to posting my suggestion to this issue.

https://discord.com/channels/179641883222474752/1168216953924624464/1204124899216728104

The TripSit chart has several different substances that are used both recreationally and medically. A good example of this is Amphetamines, which both is used recreationally and medically. SSRI's are (in 99% of cases) not use recreationally. And a drug such as Cocaine is generally taken recreationally. Of course, these vary a lot.

From my understanding of the chart, there is a position of caution when it comes to combination guidelines. SSRI's + DXM is a good example of this. While generally, any user on an SSRI is not at risk of taking 15mg of DXM, they are much more likely to be at risk when it comes to recreational dosages. The chart is used generally for recreational drug users / psychonauts (etc) and the users would most likely be following it for recreational dosages.

DPH has a bit of an issue. It seems that for certain drugs at medical dosages, the interaction of the substances will be reasonably safe, but when it comes to recreational dosages the combination should be recommended against.

While I don't recommend going back and adding this for other substances (for now) I do recommend that going forward DPH interactions are to be considered/written in the context of recreational DPH dosages and that we add a footnote on the bottom of the chart mentioning that DPH interactions are in mind for recreational and perhaps also adding that it may vary on dosages. We can aim to add notes in the combination notes so people when using the site and bot can get more information (such as saying a low dose may be safe)

**Comment by zelixir25** [comment]
I agree with this and also think maybe we should add this footnote for DXM since, as you said, someone taking Lexapro who has a DayQuil for their cold likely won't have issues but that same person taking 300mg to trip might.

**Reply by utaninja** [reply]
Agreed. I'd also like to add to this that we should note that DPH and DXM imo

Any of the other drugs on the chart that I see either has crossover for recreational dosages-medical dosages (such as amphetamine, pregabalin, ghb/gbl) or only is used 99% the time medically (such as Lithium or SSRI's)

**Comment by utaninja** [comment]
Following up to this. I'd like to thank @pre-quel for the dosage aggregation information for DPH and insight on data collection.
<img width="466" height="291" alt="image" src="https://github.com/user-attachments/assets/c2ffeb1f-1878-4bb9-b1dd-8512b5472970" />

For now, I would say a reasonable assumption for DPH recreational dosages in reference to interactions is 200-400mg.

**Comment by utaninja** [comment]
Further discussion on this here: https://github.com/TripSit/drugs/discussions/213#discussioncomment-14827583
Thank you @pre-quel and @zelixir25!
